validator/rpc/structs.go

```diff
@@ -89,3 +89,46 @@ type GetFeeRecipientByPubkeyResponse struct {
 type SetFeeRecipientByPubkeyRequest struct {
 	Ethaddress string `json:"ethaddress"`
 }
+
+// Type of key manager for the wallet, either derivedKeymanagerKind, importedKeymanagerKind, or remote.
```

Comment:
```suggestion
// Type of key manager for the wallet.
```
To avoid mismatches between the comment and actual values